src/Normalizer.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace SlimTwig;
+
+class Normalizer {
```

Comment:
Bỏ hẳn class này đi, vì nhiệm cụ của Twig ko phải là loại bỏ HTML. VD: Twig hoàn toàn có thể render 1 string dạng `<a href="{{ post.url }}">{{ post.title }}</a>`.